_posts/2017/2017-07-19-reunification.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Reunification"
+layout: "post"
+tags: ["news"]
+---
+
+Having an individual repo for each driver is causing a lot of administrative overhead so we have decided to merge all repos into a single one. This has some implications you may have to address before and after this happens so I am going to try summarize them here:
```

Comment:
Ok, will create a github issue for the purpose of people commenting about this and link it here.